homebrew/libxmlsec1.rb

```diff
@@ -0,0 +1,77 @@
+class Libxmlsec1 < Formula
```

Comment:
Is there a way we can do this not at the top level? Just having another top level folder is a bit annoying